### PR TITLE
#162951631 Build User Login API 

### PR DESCRIPTION
**Pull request by micah-akpan**
#### What does this PR do?
Have login user endpoint working. Users can now login with the right credentials
#### Description of Task to be completed?
Have the login API endpoint: /auth/login working as expected
#### How should this be manually tested?
The login API endpoint can be tested with an HTTP client, like Postman
To test the endpoint:
 - Clone the repo
 - ```npm install```
 - ```npm run dev:start```
 - Go over to Postman, open a tab, choose POST and enter the login API endpoint: /api/v2/auth/login
-  Supply the following required data (firstname, lastname, email, and password) to test successfully

#### What are the relevant pivotal tracker stories?
#162951631
#### Questions:

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -1,17 +1,23 @@
 import express from 'express';
 import logger from 'morgan';
 import helmet from 'helmet';
-import meetupAPI from './routes/meetup';
-import questionAPI from './routes/question';
-import rsvpAPI from './routes/rsvp';
+import { config } from 'dotenv';
+// import meetupAPI from './routes/meetup';
```
Comment (victor-enogwe) [inline server/app.js:5]: avoid having unused commented code

File: server/controllers/user.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      const newTableResult = await db.queryDb(createNewUserQuery);
+      return res.status(201).send({
+        status: 201,
+        data: [{
+          token: userAuthToken,
+          user: omitProps(newTableResult.rows[0], ['password'])
+        }]
+      });
+    } catch (e) {
+      res.status(400).send({
```
Comment (victor-enogwe) [inline server/controllers/user.js:47]: return missing

**Review by victor-enogwe — CHANGES_REQUESTED** [review]

File: server/controllers/user.js (hunk 2 above)
Comment (micah-akpan) [reply server/controllers/user.js:47]: Thanks, I will fix it

File: server/app.js (hunk 1 above)
Comment (micah-akpan) [reply server/app.js:5]: Okay, I will. thanks for pointing that out

**Pull request merged**